app/config/sources.inc.php

```diff
@@ -571,458 +49,1067 @@
 ];
 
 /*
-    Array structure for single website:
-    [
-       0 name,
-       1 path to local repo,
-       2 folder containing locale files,
-       3 array of supported locale,
-       4 array of supported file names,
-       5 reference locale,
-       6 url to public repo,
-       7 array of flags,
-       8 type of files (lang, raw)
-    ]
+    For each file it's possibile to specify the following fields:
+    * deadline: string, date in ISO format
+
+    * priorities: a multidimensional array, where each level of priority
+      (from 1 to 3) is associated to a list of locales. 'all' can be used to
```

Comment:
Probably good to mention if 1 is the highest or the lowest priority (since Pontoon stars, I don’t want to make any assumptions regarding priorities :p)